### ISSUE TITLE
"Review code has encountered a problem"

### ISSUE THREAD

**Issue by iStarRaw** [pmd/pmd-eclipse-plugin#64]
Hi Guys & galls,

I've not been able to correctly use the PMD in the newest version of Eclipse (see pic below). I keep getting an "javax/xml/bind/JAXBException" the "review code has encountered a problem. I tried to disable using java build path but no luck there. Somebody has an idea where to look to solve this?

Thanks!

<img width="367" alt="screen shot 2018-09-30 at 13 46 04" src="https://user-images.githubusercontent.com/35494284/46257310-b6384380-c4b7-11e8-8ecd-cc6a65a7aa5c.png">
<img width="612" alt="screen shot 2018-09-30 at 13 49 12" src="https://user-images.githubusercontent.com/35494284/46257313-bd5f5180-c4b7-11e8-99cb-8d4676364afa.png">

**Comment by adangel** [comment]
This sound like a java11 problem, where the javaee (e.g. javax.xml.bind) is not included anymore. Try to run eclipse with at most java10.

**Comment by iStarRaw** [comment]
Running with java 10 gives me this problem...

**Comment by iStarRaw** [comment]
Hi Angel,
Eclipse is for sure on java 10 (stated that in the .ini file and checked it under about eclipse/configuration) but in the error log that I get when trying to run pmd it says java 11....so where does pmd take the version from? tnx in advance for your time!
Buen domingo!

**Comment by adangel** [comment]
Do you have `--add-modules=ALL-SYSTEM` in your eclipse.ini? (javax.xml.bind is a separate module, which is deprecated and is removed with java11, but even with java10 you need to enable it).

Also please note, that you need the latest snapshot of this plugin to run it under the latest eclipse 4.9 (SimRel 2018-09).

**Comment by iStarRaw** [comment]
tnx for the quick reply :-)
Yes, I do. But to be sure, where do you want this to appear. I have the add modules all system several times.
the plugin is set to upload from (see pic). 
<img width="601" alt="screen shot 2018-09-30 at 17 53 40" src="https://user-images.githubusercontent.com/35494284/46259460-1c35c280-c4da-11e8-8811-1e578f016adb.png">
Suppose the latest version is there as well? or do I specifically need to refer to snapshot/updates?

Just to be complete, here's the pom.xml as well.
<img width="394" alt="screen shot 2018-09-30 at 17 57 53" src="https://user-images.githubusercontent.com/35494284/46259490-70d93d80-c4da-11e8-8fe2-d0e5b9f801f0.png">

**Comment by iStarRaw** [comment]
Here's the log where it states I am using java 11 but in about eclipse and in my eclipse.ini I have java 10...
[pmdLog.log](https://github.com/pmd/pmd-eclipse-plugin/files/2431722/pmdLog.log)

**Comment by adangel** [comment]
You'll need a separate update url for the snapshots: https://dl.bintray.com/pmd/pmd-eclipse-plugin/snapshots/updates/

And you need to use Java10 - you are still using java11:

```
eclipse.buildId=4.9.0.I20180906-0745
java.version=11
java.vendor=Oracle Corporation
```

I see, you are using MacOSX - I don't know how this works, but usually, you need to set the `JAVA_HOME`  and your `PATH` environment variables to make sure, you are using the correct java version.

**Comment by iStarRaw** [comment]
Ok, think were getting there. I am using java 11 on my operating system but java 10 in ECLIPSE as you can see on this screenshot. The java 11 you copy is from the PMD error log and thus is probably then about the java version my operating system is using...
will continue this weekend to see if I can get it running. will let you know what resolved it in the end, or not ;-)
<img width="569" alt="screen shot 2018-10-01 at 18 10 47" src="https://user-images.githubusercontent.com/35494284/46301093-a6972880-c5a5-11e8-9d30-2f1baa0dc8b8.png">

**Comment by adangel** [comment]
I can confirm the behavior with java11 - I get the same error. And some plugins don't even start at all.

Your eclipse.ini looks fine - I've tested it (under linux), and even when I used java11 (JAVA_HOME, PATH is set to java11), eclipse would use the Java version configured with the "-vm" option.

**Comment by adangel** [comment]
@iStarRaw I believe this is fixed via #73 . Please update and test again. If it doesn't work, please open a new issue. Thanks!
